### ISSUE TITLE
Configuration

### ISSUE THREAD

**Issue by Marketingguru** [seth-c-stenzel/PyFacebook-Personal-Scraper#1]
Hello Seth! This was just what I was looking for! Thank you so much for sharing your code! I can't seem to figure out how to get this to work, would you mind giving me some information on how to set it up? I have it connected to MySQL but it doesn't seem to want to grab the data off from the post.

**Comment by seth-c-stenzel** [comment]
Ill be honest I'm pretty fresh at coding, and hadn't gotten all the bugs worked out yet. Though I was able to get the data into a database at least to work with.

Can you tell me what errors you are getting or what issues you are having? There is a good bit of setup on the facebook side of this as well, have you registered your app with facebook, and have your auth keys?

**Comment by Marketingguru** [comment]
Yes, I did all of that. I was trying to pull the comments and likes from a
dark post, which was an advertisement. I got the
facebook-company-scraper.py to work but it doesn't save the data anywhere.
PyFace-Scraper.py was giving me an error after I entered my user token,

Traceback (most recent call last):
  File "PyFace-Scraper.py", line 199, in <module>
    main()
  File "PyFace-Scraper.py", line 192, in main
    post_id_pull(base_graph_url, posts_graph_url, request_range_full,
post_fields_request_params, access_token)
  File "PyFace-Scraper.py", line 139, in post_id_pull
    for id_key in post_requested_id_data.json()["data"]:
KeyError: 'data'
